apps/web/features/settings/components/SettingsGroup.tsx

```
@@ -49,11 +50,10 @@ function SettingPreview({
 	)
 }
 
-export function SettingsGroup({ group, values, onChange, disabled = false }: SettingsGroupProps) {
+export function SettingsGroup({ group, values, onChange, disabled = false }: props) {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'SettingsGroup'. Consider to delete it or rename it.

'SettingsGroup' is defined here:


(lint/suspicious/noRedeclare)